avail-intellij-plugin/src/main/kotlin/org/availlang/intellij/plugin/module/AvailProjectTemplate.kt

```diff
@@ -0,0 +1,185 @@
+    AvailProjectV1(
+      config.fileName,
+      true,
+      AvailRepositories(rootNameInJar = null),
+      localSettings
+    ).apply {
+      File(config.projectLocation).mkdirs()
+      val rootsDir = config.rootsDir
+      val rootName = config.rootName
+      val rootLocationDir = rootsLocationDir(rootsDir, rootName)
+      File("${config.projectLocation}/$rootLocationDir").mkdirs()
+
+      val rootConfigDir = AvailEnvironment.projectRootConfigPath(
+        config.fileName,
+        rootName,
+        config.projectLocation)
+      val root = availProjectRoot(rootConfigDir, config, rootName, rootLocationDir)
+      addRoot(root)
+      optionallyInitializeConfigDirectory(rootConfigDir)
+      config.selectedLibrary?.let { lib ->
+        configureAvailStdLib(config, lib)
+      }
+
+      updateBackingProjectFile(projectFilePath, config)
+    }
+  }
+
+  private fun AvailProjectV1.updateBackingProjectFile(
+    projectFilePath: String,
+    config: Config
+  ) {
+    if (projectFilePath.isNotEmpty()) {
+      // Update the backing project file.
+      val writer = JSONWriter.newPrettyPrinterWriter()
+      writeTo(writer)
+      File(projectFilePath).writeText(writer.contents())
+      config.environmentSettings.add(this, projectFilePath)
+    }
+  }
+
+  private fun AvailProjectV1.configureAvailStdLib(
+    config: Config,
+    lib: File
+  ) {
+    val libName = config.libraryName ?: AVAIL_STDLIB_ROOT_NAME
+    val libConfigDir = AvailEnvironment.projectRootConfigPath(
+      config.fileName, libName, config.projectLocation
+    )
+    optionallyInitializeConfigDirectory(libConfigDir)
+    val jar = AvailArtifactJar(lib.toURI())
+
+    val rootManifest = jar.manifest.roots[AVAIL_STDLIB_ROOT_NAME]
+    var sg =
+      importStyles(config, jar)
+    var tg =
+      importTemplates(config, jar)
+    val extensions = mutableListOf<String>()
+    var description = ""
+    rootManifest?.let {
+      sg = it.styles
+      tg = it.templates
+      extensions.addAll(it.availModuleExtensions)
+      description = it.description
+    }
+    val stdLib = AvailProjectRoot(
+      libConfigDir,
+      config.projectLocation,
+      libName,
+      AvailLibraries(
+        "org/availlang/${lib.name}",
+        Scheme.JAR,
+        AVAIL_STDLIB_ROOT_NAME
+      ),
+      LocalSettings(
+        AvailEnvironment.projectRootConfigPath(
+          config.fileName,
+          libName,
+          config.projectLocation
+        )
+      ),
+      sg,
+      tg,
+      extensions
+    )
+    stdLib.description = description
+    stdLib.saveLocalSettingsToDisk()
+    stdLib.saveTemplatesToDisk()
+    stdLib.saveStylesToDisk()
+    addRoot(stdLib)
+  }
+
+  private fun importTemplates(
+    config: Config,
+    jar: AvailArtifactJar
+  ) = if (config.importTemplates) {
+    jar.manifest.templatesFor(AVAIL_STDLIB_ROOT_NAME)
+      ?: TemplateGroup()
+  } else {
+    TemplateGroup()
+  }
+
+  private fun importStyles(
```

Comment:
Done in https://github.com/AvailLang/Avail/pull/310/commits/377e9c11de82c578bba8149a0c61fe573d53655a

Reply:
Update the comment to follow the commenting style. Something like:

```kotlin
/**
 * Imports the styles for the given Avail artifact jar.
 *
 * @param config 
 *   The [configuration][Config] object containing import options.
 * @param jar 
 *   The Avail artifact jar from which to import styles.
 * @return 
 *   The imported [styling group][StylingGroup] if importStyles is enabled in 
 *   the config object, otherwise an empty styling group.
 */
```